src/os_esp32.cc

```diff
@@ -576,8 +576,7 @@ void OS::heap_summary_report() { }
 
 #endif // def TOIT_CMPCTMALLOC
 
-static const int TOIT_IMAGE_DATA_SIZE = 1024;
-static const int TOIT_CONFIG_IMAGE_SIZE = TOIT_IMAGE_DATA_SIZE - UUID_SIZE;
+static const int TOIT_IMAGE_DATA_SIZE = 100 * 1024;
```

Comment:
Can this be a proper number like 128KB? It feels like an annoying restriction to have, but maybe it is good enough for now.